code/modules/heavy_vehicle/mech_interaction.dm

```diff
@@ -275,8 +272,6 @@
 		set_intent(I_HURT)
 		LAZYREMOVE(pilots, user)
 		UNSETEMPTY(pilots)
-		update_pilot_overlay()
-	return
```

Comment:
Return the return

Reply:
That return doesn't actually seem to do anything from what I can see

Reply:
The proc was removed by my instruction with the returns being mistakenly removed, but if this return does nothing then its removal is fine.